waltz-data/src/main/ddl/liquibase/db.changelog-1.3.xml

```diff
@@ -132,4 +132,12 @@
             </column>
         </addColumn>
     </changeSet>
+
+
+    <!-- 1725: remove trait and trait_usage-->
+    <changeSet id="20170307-1725-1"
```

Comment:
could you stick a <comment> element in pls ?    They get inserted into the generated ddl which should give us a bit more traceability if we ever need it.....